app/konnect/servicehub/manage-services.md

```diff
@@ -55,12 +55,28 @@ box to save.
 
 1. From the left navigation menu, click **Services**.
 
-2. Select a Service from the list.
+2. Select a Service from the dropdown menu.
 
-3. In the **Versions** section, click **+ New Version**.
+3. Navigate to **Versions**, and click **+ New Version**.
 
 4. Enter a version name and click **Create** to save.
 
+### Archive or Delete a Service Version
+
+1. From the left navigation menu, click **Services**.
+
+2. Select a Service from the dropdown menu.
+
+3. Navigate to **Versions**.
+
+4. Click on the version you want to archive or delete, and you'll be taken to the version detail page.
+
+5. Click on the **Actions** dropdown menu.
+
+6. Click **Delete Version** to permanently delete the Service version, or **Archive Version** to remove the Service version from use but not permanently delete it. You will not see archived versions in Service detail page versions dropdown.
+
+If you want to unarchive a version, return to the Service version detail page. Click **Actions**, and then click **Publish Version**.
```

Comment:
```suggestion
If you want to republish an archived version, return to the Service version detail page. Click **Actions**, and then click **Publish Version**.
```
Avoiding "unarchive" since it's not a regular word. "Retrieve an archived version" could also work.